applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1230,34 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+=====================
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
```

Comment:
This is a requirement for DFU, so we need a subsection under Requirements "Requirements for FOTA" with this info.

Reply:
Added a Requirements for FOTA section